### PR TITLE
LPS-167318 HR_416 SR is rendering the 3 dots Ellipsis as 'button menu'

### PR DESCRIPTION
**Pull request by Sharryxue**
https://issues.liferay.com/browse/LPS-167318
Added test fix commit for the failed POSHI tests. And the failed test can be passed on local with the fix commit.

/cc @gislayne-vitorino @pei-jung

### CODE REVIEW

**Comment by Sharryxue** [comment]
No related issue found in [ci:test:relevant](https://github.com/liferay-usm/liferay-portal/pull/878#issuecomment-1323498200).

**Comment by pei-jung** [comment]
Thanks for the test fix, @Sharryxue! 👍

File: modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/servlet/taglib/util/AddressActionDropdownItemsProvider.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+					).buildString());
+				dropdownItem.setLabel(
+					LanguageUtil.get(_httpServletRequest, "edit"));
+			}
+		).add(
+			dropdownItem -> {
+				dropdownItem.setHref(
+					PortletURLBuilder.create(
+						PortletURLUtil.clone(
+							_getEditAddressActionURL(), _renderResponse)
```
Comment (drewbrokke) [inline modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/servlet/taglib/util/AddressActionDropdownItemsProvider.java:83]: Same idea here as with https://github.com/liferay-usm/liferay-portal/pull/879/files#r1029735285

**Review by drewbrokke — CHANGES_REQUESTED** [review]

**Comment by pei-jung** [comment]
Closing this PR as the commits here are included in https://github.com/liferay-usm/liferay-portal/pull/879.

**Pull request closed without merging**